metrics/stackdriver_exporter.go

```diff
@@ -80,6 +116,41 @@ func newStackdriverExporter(config *metricsConfig, logger *zap.SugaredLogger) (v
 	return e, nil
 }
 
+// getStackdriverExporterOptions creates client options for the opencensus Stackdriver exporter from the given stackdriverClientConfig.
+// On error, an empty array of client options is returned.
+func getStackdriverExporterOptions(sdconfig *stackdriverClientConfig) ([]option.ClientOption, error) {
```

Comment:
I think you mean `getStackdriverClientOptions`?

Reply:
Pending